docs/src/main/sphinx/installation/cli.rst

```diff
@@ -160,6 +160,14 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Autosuggestion
+--------------
+As you type commands in CLI , you will see a completion offered after the cursor
+in a muted gray color based on history.
+
+If you press :kbd:`→` key it will accept the suggestion, replacing the contents of the command line buffer with the suggestion.
+
```

Comment:
What happens if you clear command history? I assume the completion suggestions dry up until more history builds up?

Reply:
@Ordinant I confirmed. If you clear command history no suggestions will be offered until new history builds up